Project.toml

```diff
@@ -11,6 +11,7 @@ DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
 FunctionWrappers = "069b7b12-0de2-55c6-9aab-29f3d0a68a2e"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

Comment:
Let's not add a Plots dependency -- that is quite heavy and could slow package loading quite a bit. 

I'd suggest moving the plotting utils to the tests/spatial folder for now. If we want to create simple plotting routines we can move those to a separate repo down the line.